SU-SWS/stanford_story_page
# HSDO-950 Added section header to homepage template

# READY FOR REVIEW

# Summary
- Changed homepage template to use section header instead of callout text.
- decreased the image sizes to trim the repo size a little.

# Needed By (Date)
- 8/3

# Urgency
- Low

# Steps to Test

1. Checkout branch & latest of [stanford_paragraph_types](https://github.com/SU-SWS/stanford_paragraph_types)
2. enable stanford_paragraph_types_section_header & stanford_story_page module
3. ensure the features are in default state
4. create stanford story page content and select the 'Homepage' template
5. check to make sure there are some section divider paragraphs in it.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
